docs/data/base/components/menu/UnstyledMenuBasic/css/index.js

```diff
@@ -1,87 +1,25 @@
 import * as React from 'react';
 import Menu from '@mui/base/Menu';
 import MenuItem, { menuItemClasses } from '@mui/base/MenuItem';
-import Button, { buttonClasses } from '@mui/base/Button';
+import MenuButton from '@mui/base/MenuButton';
+import Dropdown from '@mui/base/Dropdown';
 import { useTheme } from '@mui/system';
-import { ListActionTypes } from '@mui/base/useList';
 
 export default function UnstyledMenuSimple() {
```

Comment:
Shouldn't we replace the "Unstyled" prefixes for "Base"? Or are we keeping the "Unstyled"? I think it might be confusing. Maybe just use "MenuSimple"

Reply:
Yeah, good point. I'll create a separate PR for this.